proposals/lip-0043.md

```diff
@@ -128,11 +128,13 @@ The `ownName` property sets the name of the sidechain in its own state according
 | `COMMAND_ID_MAINCHAIN_REG`            | uint32 | 1                | Command ID of mainchain registration command.     |
 | `CROSS_CHAIN_COMMAND_ID_REGISTRATION` | uint32 | 0                | Cross-chain command ID of chain registration CCM. |
 | **General Constants**                 |        |                  |                                                   |
-|`MIN_FEE_PER_BYTE`                     | uint64 | 1000             | Minimum fee per byte.                             |
-|`REGISTRATION_FEE`                     | uint64 | 1000000000       | Fee to pay for a sidechain registration command.  |
-|`MAX_NUM_VALIDATORS`                   | uint32 | 199              | Maximum number of validators in a sidechain.      |
-|`MAX_LENGTH_NAME`                      | uint32 | 40               | Maximum allowed name for a sidechain.             |
-|`TAG_CHAINREG_MESSAGE`                 | bytes  | ASCII encoded string `“LSK_CHAIN_REGISTRATION_”`                     |
+| `REGISTRATION_FEE`                    | uint64 | 1000000000       | Fee to pay for a sidechain registration command.  |
+| `MAX_NUM_VALIDATORS`                  | uint32 | 199              | Maximum number of validators in a sidechain.      |
+| `MAX_LENGTH_NAME`                     | uint32 | 40               | Maximum allowed name for a sidechain.             |
+| `MAX_UINT32`                          | uint64 | 4294967295      | Maximum value for a 32-bit unsigned integer.      |
+| `MAX_UINT64`                          | uint64 | 18446744073709551615 | Maximum value for a 64-bit unsigned integer.  |
+| `THRESHOLD_MAINCHAIN`                 | uint32 | `68`             |The certificate threshold used for Lisk mainchain. |
+| `TAG_CHAINREG_MESSAGE`                | bytes  | ASCII encoded string `“LSK_CHAIN_REGISTRATION_”`.                    |
```

Comment:
Sorry to bring this up now, but shouldnt this be `TAG_CHAIN_REG_MESSAGE`?

Reply:
modified

Reply:
modified